cloudify_agent/api/pm/base.py

```diff
@@ -107,9 +101,47 @@ class Daemon(object):
         the password for the broker connection
         defaults to 'guest'
 
-    ``manager_port``:
+    ``file_server_host``:
+
+        the IP or hostname of the file server (Required)
+
+    ``rest_host``:
```

Comment:
if you expose the rest host and the rest protocol, why are you only exposing the file server host and not its protocol?